packages/dialtone-vue2/components/hovercard/hovercard.test.js

```diff
@@ -0,0 +1,147 @@
+import { mount } from '@vue/test-utils';
+import DtHovercard from './hovercard.vue';
+
+const MOCK_DEFAULT_SLOT_MESSAGE = 'Message';
+const MOCK_HEADER_CONTENT = 'Hovercard Title';
+const MOCK_FOOTER_CONTENT = 'Hovercard Footer';
+
+const baseProps = { id: 'hovercard-1' };
+const baseAttrs = {};
+const baseSlots = {
+  content: MOCK_DEFAULT_SLOT_MESSAGE,
+  headerContent: MOCK_HEADER_CONTENT,
+  footerContent: MOCK_FOOTER_CONTENT,
+};
+
+const baseScopedSlots = {
+  anchor: '<template #anchor="{ attrs }">' +
+          '<button data-qa="dt-button" v-bind="attrs">Hover me</button>' +
+          '</template>',
+};
+
+let mockProps = {};
+let mockAttrs = {};
+let mockSlots = {};
```

Comment:
these are not being used, it could be removed, up to you

Reply:
Fixed in latest changes